test/input.txt

```diff
@@ -54,6 +54,16 @@
   # should not allow adding duplicate persons
   add Esther Potato p/555555 e/esther@not.a.real.potato pa/555, epsilon street t/tubers t/starchy
 
+##########################################################
+# test add multiple people command
+##########################################################
+
+  # Should show that all persons are duplicate persons
```

Comment:
Having test cases that cover valid addition of new persons using your command would be useful.